modules/preparing-an-initial-cluster-deployment-for-mce-disconnected.adoc

```diff
@@ -5,9 +5,9 @@
 :_mod-docs-content-type: PROCEDURE
 [id="preparing-an-initial-cluster-deployment-for-mce-disconnected_{context}"]
 
-= Preparing an Agent-based cluster deployment for the multicluster engine for Kubernetes operator while disconnected
+= Preparing an Agent-based cluster deployment for the {mce} while disconnected
 
-You can mirror the required {product-title} container images, the multicluster engine for Kubernetes operator, and the Local Storage Operator (LSO) into your local mirror registry in a disconnected environment.
+You can mirror the required {product-title} container images, the {mce-short}, and the Local Storage Operator (LSO) into your local mirror registry in a disconnected environment.
 Ensure that you note the local DNS hostname and port of your mirror registry.
```

Comment:
If you wanted to separate this sentence into a separate paragraph, this did not work. If you did not want to make such a change, please ignore this comment. (I'm just not 100% sure what GitHub is showing me here)